src/utils/shared.test.ts

```diff
@@ -13,6 +13,22 @@ describe("getChainId for v2 document", () => {
     expect(getChainId(document)).toStrictEqual(5);
   });
 
+  it("should return the correct chainId for polygon mumbai network", () => {
+    const document = {
+      ...invoiceV2,
+      data: { ...invoiceV2.data, network: { chain: "MATIC", chainId: "80001" } },
+    } as unknown as WrappedOrSignedOpenAttestationDocument;
```

Comment:
the test shouldn't pass since MATIC is 137 no?

Reply:
nono, this matic is  the block chain, either ethereum: ETH or polygon: MATIC.

Reply:
then the 80001 is testnet mumbai, 137 is polygon mainnet